src/core/tests/Test_CoreMain.py

```diff
@@ -1004,8 +1004,8 @@ def test_auto_assessment_resets_maintenance_run_id_and_health_store_id_to_None(s
         CoreMain(argument_composer.get_composed_arguments())
 
         # check values of health_store_id and maintenance_run_id
-        self.assertEquals(runtime.execution_config.health_store_id, None)
-        self.assertEquals(runtime.execution_config.maintenance_run_id, None)
+        self.assertEqual(runtime.execution_config.health_store_id, None)
+        self.assertEqual(runtime.execution_config.maintenance_run_id, None)
```

Comment:
From this PR - https://github.com/Azure/LinuxPatchExtension/pull/306 
(Py 3.12 failure. That PR should ideally go in first.)